### PR TITLE
Upgrades and Changes

### PR DESCRIPTION
**Pull request by ukrocks007**
NPM Package upgraded & frozen
Changes for Publisher API [Using API key & Bulk API body structure]

### CODE REVIEW

**Review by deepakprabhakara — APPROVED** [review]

File: src/test.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import * as Retraced from "./";
       projectId: "aaaahahahahahahaha",
     });
 
-    expect(garbo).to.exist;
+    console.log(expect(garbo).to.exist);
```
Comment (deepakprabhakara) [inline src/test.ts:13]: remove console.log

**Review by deepakprabhakara — APPROVED** [review]

File: src/test.ts (hunk 1 above)
Comment (ukrocks007) [reply src/test.ts:13]: There were linking errors so I added logs at the end of tests

**Review by deepakprabhakara — APPROVED** [review]

File: src/test.ts (hunk 1 above)
Comment (deepakprabhakara) [reply src/test.ts:13]: linking errors as in?

Comment (ukrocks007) [reply src/test.ts]: Lint*

Comment (deepakprabhakara) [reply src/test.ts]: aah, let me check.

**Pull request merged**